app/components/embed/media_preview_image_component.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+module Embed
+  # Displays an image (e.g. photo of the media) for a media resource
+  class MediaPreviewImageComponent < ViewComponent::Base
+    include StacksImage
+
+    # @param [String] druid the object identifier
+    # @param [ResourceFile] file the file to display
+    # @param [Integer] file_index the offset of this file in the resource
```

Comment:
`s/file_index/resource_index/` ?

Reply:
And not sure if the rest of the verbiage also needs changing.

Reply:
Thanks. Fixed that up.